script/8_deploySwapAndBridge.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+echo "Instructing the shell to exit immediately if any command returns a non-zero exit status..."
+set -e
+echo "Done."
+
+echo "Navigating to the root directory of the project..."
+cd ../
+echo "Done."
+
+echo "Setting environment variables..."
+source .env
+echo "Done."
+
+echo "Creating $NETWORK directory inside deployment directory..."
+if [ -z "$NETWORK" ]
+then
+      echo "NETWORK variable inside .env file is not set. Please set NETWORK environment variable."
+      exit 1
+else
+      if [ -d "deployment/swap_and_bridge/$NETWORK" ]
+      then
+            echo "Directory deployment/swap_and_bridge/$NETWORK already exists."
+      else
+            mkdir -p deployment/swap_and_bridge/$NETWORK
+      fi
+fi
+echo "Done."
+
+echo "Deploying and if enabled verifying SwapAndBridge smart contract for DIVA protocol..."
+forge script --rpc-url="$L1_RPC_URL" --broadcast --verify --verifier blockscout --verifier-url $L1_VERIFIER_URL -vvvv script/swap_and_bridge/contracts/SwapAndBridge.s.sol:SwapAndBridgeScript $L1_STANDARD_BRIDGE_ADDR $L1_TOKEN_ADDR_DIVA $L2_TOKEN_ADDR_DIVA --sig 'run(address,address,address)' 
+echo "Done."
+
+echo "Deploying and if enabled verifying SwapAndBridge smart contract for LIDO protocol..."
+forge script --rpc-url="$L1_RPC_URL" --broadcast --verify --verifier blockscout --verifier-url $L1_VERIFIER_URL -vvvv script/swap_and_bridge/contracts/SwapAndBridge.s.sol:SwapAndBridgeScript  $L1_LIDO_BRIDGE_ADDR $L1_TOKEN_ADDR_LIDO $L2_TOKEN_ADDR_LIDO --sig 'run(address,address,address)'
```

Comment:
Probably I'm missing something, but where is `$L2_TOKEN_ADDR_LIDO` deployed?
We deploy `$L2_TOKEN_ADDR_DIVA` inside `7_deployWdivETHContract.sh` script.
I can't find where is Lido deployed on L2?

What about L1 networks deployment for Lido and Diva? Were they deployed by Lido and Diva teams themself?

Reply:
Regarding L1 deployments: they are deployed by Diva and Lido team, we have no control over that.

Reply:
Ok, cool!
Should we thenupdate `.env.mainnet` if we already know what's the address of L1 deployments on L1 Mainnet?

Reply:
These integrations have not been deployed on mainnet yet. Once that's done we can update the env file

Reply:
Ok, nice!

What about this part for L2?
```
Probably I'm missing something, but where is $L2_TOKEN_ADDR_LIDO deployed?
We deploy $L2_TOKEN_ADDR_DIVA inside 7_deployWdivETHContract.sh script.
I can't find where is Lido deployed on L2?
```

Reply:
It's deployed in another feature, related but independent from this. The deployment is rather complex and we decided to track it independently for clarity and audit purposes. From the POV of this feature, it's just a parameter in the .env file

Reply:
Thanks for clarification!